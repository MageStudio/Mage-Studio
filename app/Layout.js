import React from 'react';
import { Row, Col, Layout } from 'antd';
//const { Header, Footer, Sider, Content } = Layout;
import Header from './header/Header';
import Footer from './footer/Footer';
import Sidebar from './sidebar/Sidebar';
import Scene from './scene/Scene';

import RightSidebar from './RightSidebar/RightSidebar';

import './style.scss';

export default (props) => (
    <div className="app">
        <Header />
        <Row className="main-container">
            <Sidebar />
            <div className="canvas-container">
                <Scene />
            </div>
            <RightSidebar />
        </Row>
        <Footer />
    </div>
);
