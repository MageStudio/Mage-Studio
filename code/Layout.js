import React from 'react';
import {Row} from 'antd';

import Header from '../lib/header/Header';
import Footer from '../lib/footer/Footer';
import CodeEditor from './CodeEditor';

import '../lib/style.scss';
import './editor.scss';

export default () => {
    return (
        <div className="app">
            <Header />
            <Row className="main-container code-editor">
                <CodeEditor />
            </Row>
            <Footer />
        </div>
    );
};
