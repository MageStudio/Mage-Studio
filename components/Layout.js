import React from 'react';
import { Row, Col, Layout } from 'antd';
//const { Header, Footer, Sider, Content } = Layout;
import Header from './header/Header';
import './style.scss';

export default (props) => (
    <div className="app">
        <Header />
        <Row className="main">
            <Col span={8}>col-8</Col>
            <Col span={16}>{props.children}</Col>
        </Row>
        <Row className="footer">
            <Col span={12}>col-6</Col>
        </Row>
    </div>
);
