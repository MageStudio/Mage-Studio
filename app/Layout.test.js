import React from 'react';
import { shallow } from 'enzyme';

import Layout from './Layout';

describe('Layout', () => {

    it('should render fine', () => {
        const component = shallow(<Layout />);
        expect(component).toMatchSnapshot();
    });
});
