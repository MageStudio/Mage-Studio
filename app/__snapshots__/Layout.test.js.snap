// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render fine 1`] = `
<div
  className="app"
>
  <Connect(Header) />
  <Row
    className="main-container"
    gutter={0}
  >
    <LeftSidebar />
    <div
      className="canvas-container"
    >
      <Connect(SceneContainer) />
    </div>
    <Connect(RightSidebar) />
  </Row>
  <Footer />
  <Connect(ProjectModal) />
</div>
`;
